main.css

```diff
@@ -100,7 +100,7 @@ button {
     color: var(--white);
     width: 100%;
     bottom: 0;
-    position: relative;
+    position: fixed;
```

Comment:
Weird.
I tried the same previously but didn't work for me and now it seems to be working fine. I guess when I tried this, other css classes may have different values. Anyway, thanks for fixing this. this was bugging me for a long time